src/services/mongoose/schema.ts

```diff
@@ -315,16 +280,90 @@ const UserSchema = new Schema<UserRecord>(
       type: String,
       index: true,
     },
+
+    // TODO: delete post migration
+    phone: {
+      type: String,
+      index: true,
+      unique: true,
+      sparse: true,
+    },
+    twilio: {
+      carrier: {
+        error_code: String, // check this is the right syntax
+        mobile_country_code: String,
+        mobile_network_code: String,
+        name: String,
+        type: {
+          types: String,
+          enum: ["landline", "voip", "mobile"],
+        },
+      },
+      countryCode: String,
+    },
+    deviceToken: {
+      type: [String],
+      default: [],
+    },
+    language: {
+      type: String,
+      enum: [...Languages, ""],
+      default: "",
+    },
+    // END TODO
   },
   { id: false },
 )
 
-UserSchema.index({
+AccountSchema.index({
   title: 1,
   coordinates: 1,
 })
 
-export const User = mongoose.model<UserRecord>("Account", UserSchema)
+export const Account = mongoose.model<AccountRecord>("Account", AccountSchema)
+
+const UserSchema = new Schema(
+  {
+    userId: {
+      type: String,
+      required: true,
+      index: true,
+    },
+    language: {
+      type: String,
+      enum: [...Languages, ""],
+    },
+    deviceTokens: {
+      type: [String],
+      default: [],
+    },
+    phoneMetadata: {
+      type: {
+        carrier: {
+          error_code: String, // check this is the right syntax
```

Comment:
Does this need checking pre-merge?

Reply:
I don't think so. this code was copy/paste: https://github.com/GaloyMoney/galoy/pull/1861/files/33b17223db559cb9b1dc8d5e49df8d7b53cd5ec3#diff-3e0c24fa2039e07c9e45083c6d45ecebbbc4d31d9133de31e215ecb31e52996cL202

There are new element that twilio recently added, so probably a good thing to dive into while reviewing this TODO: https://www.twilio.com/blog/detect-sim-swap-sms-otp-2fa